src/ResourceManagement/Logic/Logic.Tests/ScenarioTests/IntegrationAccountSessions.ScenarioTests.cs

```diff
@@ -66,15 +66,12 @@ public void CreateAndDeleteIntegrationAccountSession()
                     integrationAccountName: integrationAccountName,
                     sessionName: integrationAccountSessionName);
 
-                // Getting an absent record should not throw, RP returns 404 and error response
-                var errorResponse = client.Sessions.Get(
-                    resourceGroupName: Constants.DefaultResourceGroup,
-                    integrationAccountName: integrationAccountName,
-                    sessionName: integrationAccountSessionName) as ErrorResponse;
-                Assert.NotNull(errorResponse);
-                Assert.NotNull(errorResponse.Error);
-                Assert.Equal(expected: "SessionNotFound", actual: errorResponse.Error.Code);
-                Assert.Contains(expectedSubstring: "could not be found in integration account", actualString: errorResponse.Error.Message);
```

Comment:
@daviburg why were all of these asserts removed? Does the added assert below ensure there is no regression with the above asserts being removed?